Ginger/Ginger/AutomatePageLib/AddActionMenu/ApplicationModels/POMNavPage.xaml.cs

```diff
@@ -99,11 +99,32 @@ public POMNavPage(Context context)
 
             mContext.PropertyChanged += MContext_PropertyChanged;
             mPOMPage.OnSelect += MainTreeView_ItemSelected;
+
+            //mPOMPage.xTreeView.xTreeViewTree.ItemDoubleClick += POMItemsTreeView_ItemDoubleClick;
+
             SetElementsGridView();
             xPOMFrame.Content = mPOMPage;
             xPOMSplitter.Visibility = Visibility.Collapsed;
         }
 
+        private void POMItemsTreeView_ItemDoubleClick(object sender, EventArgs e)
+        {
+            if (sender is TreeViewItem)
+            {
+                ApplicationPOMTreeItem apiModelTreeItem = (sender as TreeViewItem).Tag as ApplicationPOMTreeItem;
+                if (apiModelTreeItem != null)
+                {
+                    //ApplicationPOMModel selectedPOM = apiModelTreeItem.NodeObject() as ApplicationPOMModel;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4507606)